pkg/numaresourcesscheduler/manifests/yaml/configmap.nrt.yaml

```diff
@@ -22,7 +22,6 @@ data:
               - name: NodeAffinity
               - name: NodePorts
               - name: NodeResourceTopologyMatch
-              - name: NodeResourcesFit
```

Comment:
no, this is wrong. The overlap is not complete, we need to keep them both.

Reply:
Yeah, we need to have both the plugins enabled.